django/test/testcases.py

```diff
@@ -928,6 +917,16 @@ def _fixture_teardown(self):
                          allow_cascade=self.available_apps is not None,
                          inhibit_post_migrate=inhibit_post_migrate)
 
+            # If there was some initial data from migrated apps, we should restore them for next tests.
```

Comment:
", restore them for the next test." (chop "we", add "the") -- wrap comment at 79 characters.